app/src/main/res/layout/activity_create_new_password__page.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Make the text larger like the other pages
and make the inputs match_constraint for width
and make the left and right constraints set to 30
***do this match_constraint width and left and right constraints change for all the input fields in all three pages you made!***
Make the button red
Make the inputs and such closer together similar to this:
![image](https://user-images.githubusercontent.com/38587739/67540697-514c1e80-f6b4-11e9-9986-7dd89fc52328.png)